common/app/routes/challenges/redux/completion-saga.js

```diff
@@ -40,14 +40,7 @@ function submitModern(type, state) {
   const { tests } = state.challengesApp;
   if (tests.length > 0 && tests.every(test => test.pass && !test.err)) {
     if (type === types.checkChallenge) {
-      return Observable.of(
-        makeToast({
-          message: `${randomCompliment()} Go to your next challenge.`,
-          action: 'Submit',
-          actionCreator: 'submitChallenge',
-          timeout: 10000
-        })
-      );
+      return Observable.just(null);
```

Comment:
Removing this observable completely means a `Keep Trying` toast appearing on every `checkChallenge`